### ISSUE TITLE
A11y: The aria-expanded in calcite-block should be on the button not the article element 

### ISSUE THREAD

**Issue by kellyhutchins** [Esri/calcite-components#1917]
### Summary
This issue was reported during an accessibility audit of the configurable apps. Here's how they reported the issue: 

The info panel's points of interest expandable/collapsible controls contain invalid ARIA attributes and are not marked up correctly:
-Invalid: the aria-expanded attribute is on the article element
-The button is missing aria-expanded
The aria-expanded attribute should be set on the <button> elements.

**Comment by jcfranco** [comment]
Installed.

**Comment by kellyhutchins** [comment]
Tested locally and looks great. Thanks for the quick fix @jcfranco
